src/index.html

```diff
@@ -1,17 +1,204 @@
 <!doctype html>
 <html lang="en">
+
   <head>
```

Comment:
```suggestion
<html lang="en">
  <head>
```
Don't add newlines between parent and child elements